src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,129 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.UserNotAllowed;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+     void setup() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+        user.setLogin("volodymyr.nyshta@gmail.com");
+        user.setAge(21);
+        user.setPassword("123Test");
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        User actual = new User();
+        actual.setAge(20);
+        actual.setPassword("asdfasdf");
+        actual.setLogin("testtesttest@gmail.com");
+        User registeredUser = registrationService.register(actual);
+        assertEquals(registeredUser, actual);
```

Comment:
The test method name `register_validUser_Ok` does not follow the naming convention suggested in the checklist. It should be named `register_validUser_ok` to maintain consistency.